app/views/profiles/_form.html.slim

```diff
@@ -11,13 +12,20 @@ h2 = t('.profile')
   = f.input :about_me
   = f.input :time_zone, as: :time_zone, priority: /#{Time.zone.name}/, include_blank: false
   = f.association :subjects
+
+  = f.simple_fields_for :work_experiences do |work_experience|
+    = render 'work_experience_fields', f: work_experience
+  .links
+    = link_to_add_association t('.add_work_experience'), f, :work_experiences
```

Comment:
Since every nested here form have the same structure, is it worth DRY it up with smth like
```
  - %i[work_experiences educations certifications].each do |profile_nested|
    = f.simple_fields_for profile_nested do |profile_nested_fields|
      = render "#{profile_nested}_fields", f: profile_nested_fields
    .links
      = link_to_add_association t(".add_#{profile_nested}"), f, profile_nested
```
or even move nested form to the partial and do it like
`= render partial: 'profile_nested_form', collection: %i[work_experiences educations certifications]`